recipes/eo-learn/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "eo-learn" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ff756e5098f8addd94f9fa2421cb7a96ea1810bb9e89f1750dddb611b67ab329
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
```

Comment:
As mentioned before, I think you only need `pip` and `python >=3.5` in the `host` section. Can you please try adding the other dependencies in the `run` section?

Reply:
The recipe looks good otherwise, so we can merge it after you try the suggested modification.